lms/lib/comment_client/user.py

```diff
@@ -113,7 +113,12 @@ def subscribed_threads(self, query_params={}):
             metric_tags=self._metric_tags,
             paged_results=True
         )
-        return response.get('collection', []), response.get('page', 1), response.get('num_pages', 1)
+        return (
+            response.get('collection', []),
+            response.get('page', 1),
+            response.get('num_pages', 1),
+            response.get('thread_count'),
+        )
```

Comment:
`active_threads` is being used in a different flow and only from web views. Web view was not making use of `result count` field as it has been added to forums services as part of this PR when we needed to change mobile APIs to return total result count in paginated endpoints.

We can make this change available to web views too; but probably we should take an insight on it from someone in web team. What do you say?

Also, the fact that we are creating APIs for discussion module makes me worry if we are going to keep two separate implementations for same module based on client. I myself would like to make discussion APIs as a standalone implementation for every kind of client (i.e. Web or Mobile). May be not now as mobile APIs are in progress and are not covering everything that has been done in web discussion forum so far but may be in future?